proposals/csharp-10.0/parameterless-struct-constructors.md

```diff
@@ -0,0 +1,106 @@
+# Parameterless struct constructors
+
+## Summary
+
+Support explicit parameterless constructors for value types.
```

Comment:
```suggestion
Support parameterless constructors for value types.
```

In the case where you have a field initializer the parameterless constructor is implicitly defined